File: Lesson13_1/Animal.java
package Lesson13_1;

public abstract class Animal {
    protected String name;
    protected int age;

    public abstract void voice();

    public abstract void feed(int hunger);

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }
}

File: Lesson13_1/Animal.py
from abc import ABC, abstractmethod

class Animal(ABC):
    
    @abstractmethod
    def voice(self):
        pass

    @abstractmethod
    def feed(hunger):
        pass

    @abstractmethod
    def getName():
        pass

    @abstractmethod
    def setName(name):
        pass

    @abstractmethod
    def getAge():
        pass

    @abstractmethod
    def setAge(age):
        pass
